ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -495,7 +503,31 @@ def get_type_primitive_type(self, data_type):
         return data_type
 
     def get_database_service(self, service_name: str) -> DatabaseService:
-        service = self.metadata.get_by_name(entity=DatabaseService, fqn=service_name)
+        """
+        Method to get and create Database Service
+        """
+        try:
+            file_path = (
+                Path(__file__).cwd() / "examples/amundsen_data/service_type.json"
```

Comment:
I think we have handled the path to the resources differently in other parts of the code like [this](https://github.com/open-metadata/OpenMetadata/blob/a7347e1873d3b23251a02d262dcd066158bfa3c5/ingestion/src/metadata/utils/workflow_output_handler.py#L47) one.

Reply:
Here actually I am getting the value of `Path(__file__) .parent.parent` inside the env , so did these . Need to put like 10 parent to make to actual path.